src/js/state/helpers.js

```diff
@@ -492,17 +492,20 @@ export function createInitialState(formConfig) {
 
       return state;
     }, {
-      data: {
-        privacyAgreementAccepted: false,
-      },
+      data: {},
       pages: {},
     });
 
+  // Initialize the preSubmit flag if one was specified
+  const preSubmitInfo = formConfig.preSubmitInfo;
+  if (preSubmitInfo && preSubmitInfo.field) {
+    pageAndDataState.data[preSubmitInfo.field] = false;
```

Comment:
When this field had a hardwired name it could easily be initialized above. Now that the name can be configured and the field doesn't even need to be present in some cases it gets messier to try and define it in the initial object. I could have done it but it wouldn't be that readable IMO.

The only reason I initialized this field at all is because there are tests that demand a `false` initial value and I was hesitant to change existing tests. However, given my experience with the tests I'm more inclined to change them now.

What I would prefer to do is not initialize this field at all since we only really care if the value is `true` if the `formConfig` indicates it's required. The form cannot be submitted unless the value is `true`. An `undefined` or missing field is not `true` which works fine and it saves a bunch of ugly code like this.